### ISSUE TITLE
Add download for megaman_cps15.zip and update file location

### ISSUE THREAD

**Issue by movisman** [MAME-GETTER/MiSTer_MAME_SCRIPTS#25]
Hi there,

A few weeks back, Jotego added support for the Mega Man CPS1.5 hack (_alternatives):
https://github.com/jotego/jtbin/blob/master/mra/_alternatives/_Mega%20Man%20%20The%20Power%20Battle%20-CPS1/Mega%20Man%20-%20CPS1.5.mra

However, the MAME-GETTER script does not currently pull the ROM file from archive.org (megaman_cps15.zip, from here: https://archive.org/details/mmtpb.hack).

In addition, the MRA file on Jotego's repository is currently looking for megaman_cps15.zip in the 'mame' folder on MiSTer. I think the path should be changed to look for the ROM in 'hbmame', given that the ROM is a hack and not part of the official MAME romset.

So if possible - the MRA should be updated to point to the 'hbmame' folder, and hbmame-merged-set-getter.sh should be updated to download the megaman_cps15.zip from archive.org.

I was going to correct this myself and raise a PR for both, but I thought I should create an issue here first in case I am missing something or you wish to make the changes yourself.

Thanks!

**Comment by MAME-GETTER** [comment]
Here is an important point: These names of these scripts are HBMAME-GETTER and MAME-GETTER, if a rom is not officially in these set the it makes it very hard to support.  You should encourage the author of this to put this work in the HBMAME or MAME sets.

On day one this was supported with an ugly workaround. it appears that the MRA maker made a change and broke compatibility with the GETTER scripts. I am not going to keep supporting this with ugly hacks.  

When this megaman hack is added to the official HBMAME set this will fix itself.

**Comment by movisman** [comment]
Not a problem and completely understood. The only reason I asked initially, was because I had previously seen that sfzch_cps15.zip had been added manually to the HBMAME-GETTER script, pointing to a download from archive.org - so I naively thought that this could also be done for Mega Man.

Understood that this is hacky and is better to only support ROMs which have been added to the office HBMAME or MAME sets for future roms.

One thing to mention, I just noticed that on the hbmame-merged-set-getter.sh script, actually the link to the SFZ hack download via archive.org on lines 171-173 is now dead, it's been taken down. So perhaps this can be removed, along with the ROM at the very bottom of the script.

Thanks!
